src/box/lua/tuple_format.c

```diff
@@ -72,6 +70,11 @@ lbox_tuple_format_new(struct lua_State *L)
 	mpstream_flush(&stream);
 	size_t format_data_len = region_used(region) - region_svp;
 	const char *format_data = xregion_join(region, format_data_len);
+	if (mp_typeof(*format_data) != MP_ARRAY) {
+		diag_set(IllegalParams, "Wrong space format");
```

Comment:
I guess error message should be more specific. Like "format should be an array"

Reply:
Changed to "Wrong space format: expected array".